dotcom-rendering/src/client/islands/initHydration.ts

```diff
@@ -39,65 +40,39 @@ export const initHydration = async (
 	const name = getName(element);
 	const props = getProps(element);
 	const config = getConfig(element);
+	const priority = getPriority(element);
 
 	if (!name) return;
+	if (!priority) return;
+
+	const scheduleHydration = () =>
+		schedule(
+			name,
+			doHydration.bind(null, name, props, element, emotionCache, config),
```

Comment:
We do not use [`Function.prototype.bind`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) anywhere in the codebase